### PR TITLE
Support optional resource usage name field

### PR DESCRIPTION
**Pull request by vekterli**
@geirst please review

If present, will be reported alongside the resource type in the feed block description string.

### CODE REVIEW

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/hostinfo/ResourceUsage.java [hunk 1]
```diff
@@ -11,26 +11,33 @@
  */
 public class ResourceUsage {
     private final Double usage;
+    private final String name;
```
Comment (geirst) [inline clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/hostinfo/ResourceUsage.java:14]: We could consider using Optional here, but I guess that doesn't work well with the @JsonProperty annotations used in the constructor.

**Review by geirst — APPROVED** [review]
👍

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/hostinfo/ResourceUsage.java (hunk 1 above)
Comment (vekterli) [reply clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/hostinfo/ResourceUsage.java:14]: Will consider this for a later follow-up

**Comment by vekterli** [comment]
CI run seems to have failed before even starting compilation. Wasn't me!

**Pull request merged**